src/chapters/az-cli/walkthrough_az-cli.rst

```diff
@@ -0,0 +1,537 @@
+
+For each resource we provision we will use a methodical approach:
+
+- **Exploration**: issue a help command to learn about the the resource
+- **Planning**: consider our tasks and what Commands, Sub-Groups and Arguments are needed to accomplish them
+- **Provisioning**: put our Planning into action and provision the resource
+- **Configuring**: any extra steps that use the new resource to complete our tasks
+
+This is a process you can carry with you as you continue to work with the ``az CLI`` and other tooling. A good idea is to start a set of notes for easy access to common tasks you are faced with. These notes will prove invaluable when you begin composing commands together into automation scripts. 
+
+.. note::
+
+    As a cross-platform tool all of the base commands will work the same on any OS or shell. In some cases we will utilize shell-specific syntax to accomplish our tasks. Where there are syntactical differences the commands for both Linux/BASH and Windows/PowerShell will be presented.
+
+Setup
+=====
+
+Before we begin you have to log in with your Azure subscription. This command will open your browser to enter your credentials. After logging in you can close the tab and return to your terminal.
+
+.. sourcecode:: powershell
+
+    > az login
+
+Configuring Defaults
+--------------------
+
+Once you are logged in you can make use of the global ``configure`` Command to set defaults for the CLI. This will make working with resources easier by automatically inserting default values for common required Arguments. Let's set a default Azure Location for our resources.
+
+.. sourcecode:: powershell
+
+    # -d can also be used as a shorthand 
+    > az configure --default location=eastus
+
+Now any resource that has a required Argument of ``--location`` or ``-l`` will automatically have ``eastus`` set as its value.
+
+.. admonition:: note
+
+    We will use the ``configure`` Command to set defaults for common Arguments throughout this walkthrough. You can change your default values at any time by re-issuing the command with new ``name=value`` pairs. 
+    
+    You can also view your defaults using the ``--list-defaults`` Argument:
+
+    .. sourcecode:: powershell
+
+        # -l can also be used as a shorthand
+        > az configure --list-defaults
+
+Resource Groups
+===============
+
+As always we will begin by provisioning a resource group to hold the other resources associated with it. Resource groups are an organizational tool to keep track of and manage resources as a whole instead of individually. One of the benefits of this approach is that it is easy to dispose of all of the resources by deleting the resource group they are contained in instead of deleting each of them one-by-one.
+
+Exploration
+-----------
+
+Let's begin by using our trusty help command to learn what management is available for resource groups. Although it's confusing remember that resource groups are accessed under the Group name ``group``.  
```

Comment:
Suggested change from "Although it's confusing remember that resource groups are accessed under..." to "Resource groups are accessed under the az CLI Group name ``group``."

Reply:
after looking at it again, i'd be fine if you don't change it, because it is super confusing. However explicitly referring to group as an az CLI Group consistently could help clear up confusion.